src/components/bubble/Bubble.jsx

```diff
@@ -21,17 +20,43 @@ type AligmentType = 'start' | 'center' | 'end';
 
 type DirectionType = 'left' | 'right' | 'top' | 'bottom';
 
-export const ALIGNMENT = {START: 'start', CENTER: 'center', END: 'end'};
-export const DIRECTION = {
+export const ALIGNMENT: {
```

Comment:
missing here TS `as const` :) 

How about trick with `Object.freeze()`? I know right now you made it ok, but have you tested it?

Reply:
hey @matikrk, thanks!, Object.freeze is not working as expected with TS neither with flow-to-ts (I hoped that freeze will be converted to as const). I tested `as const` and it works nice.

I started from codemod for this but I decided to get back to compatible solution over codemod becuase I would like to avoid using those const at all.